package.json

```diff
@@ -57,18 +57,22 @@
     ]
   },
   "scripts": {
+    "clean": "lerna run clean; rm -rf node_modules",
     "build": "lerna run build",
+    "link-all": "lerna run link",
+    "watch": "FORCE_COLOR=true lerna run watch --parallel --stream",
     "sb:start": "start-storybook -p 6006",
     "sb:ci": "start-storybook -p 6006 --ci",
     "test": "jest",
-    "test:cover": "yarn jest --coverage",
+    "test:cover": "jest --coverage",
     "test:cypress:dev": "start-server-and-test sb:ci http://localhost:6006 cy:open",
     "test:cypress": "start-server-and-test sb:ci http://localhost:6006 cy:run",
     "cy:run": "cypress run",
     "cy:open": "cypress open",
-    "lint": "yarn eslint . --ignore-path .gitignore --ext .ts,.tsx --max-warnings=0",
-    "format": "yarn prettier --ignore-path .gitignore --write \"**/*.{tsx,ts,js,css,html,json}\"",
-    "format:check": "yarn prettier --ignore-path .gitignore --check \"**/*.{tsx,ts,js,css,html,json}\""
+    "lint": "eslint . --ignore-path .gitignore --ext .ts,.tsx --max-warnings=0",
+    "format": "prettier --ignore-path .gitignore --write \"**/*.{tsx,ts,js,css,html,json}\"",
+    "format:check": "prettier --ignore-path .gitignore --check \"**/*.{tsx,ts,js,css,html,json}\"",
+    "lint-fix": "eslint . --fix --ignore-path .gitignore --ext .ts,.tsx"
```

Comment:
can we have the two lint commands together after the format one.

Reply:
oh yea, and could we also do `lint:fix`

Reply:
fixed in next commit